### PR TITLE
Async implementation for Nobø Ecohub

### PR DESCRIPTION
**Pull request by oyvindwe**
Requires async version of https://github.com/echoromeo/pynobo

### CODE REVIEW

File: custom_components/nobo_hub/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "Nobø Hub / Nobø Energy Control",
     "documentation": "https://github.com/echoromeo/hanobo",
     "issue_tracker": "https://github.com/echoromeo/hanobo/issues",
-    "requirements": ["pynobo>=1.1.2"],
-    "version": "1.1.2",
+    "requirements": ["pynobo>=2.0.0"],
```
Comment (echoromeo) [inline custom_components/nobo_hub/manifest.json]: downgrade to 1.2.0

**Comment by oyvindwe** [comment]
This PR is ready for review now, but PR #33 should be merged first!

This PR is intended so users can get benefit from (and test) the new configuration UI while the https://github.com/home-assistant/core/pull/50913 with the official component is being reviewed.

**Comment by oyvindwe** [comment]
Note there is a design flaw in this version: the `PRESET` value (off/eco/comfort) will not be updated when all zones are in HVAC_MODE_AUTO, as there is no event from the hub to drive this change, only the clock ticking. This is addressed in the official integration.

I suggest closing this PR. I'll keep the branch in my `hanobo` fork until the official integration is released if someone wants to play with it.

**Comment by echoromeo** [comment]
Thanks, looking forward to the official integration!
At that time I will "end-of-life" this repo and recommend people to use that one.

**Pull request closed without merging**